tests-e2e/cypress/e2e/edx-frontend-school/schoolContacts.cy.ts

```diff
@@ -23,6 +23,22 @@ describe('School Contacts Page', () => {
         expect(val).to.contains(LocalDate.now().toString());
       })
     });
+
+    it('Can remove contact', () => {
+      cy.visit('/');
+      cy.get(selectors.dashboard.title, {timeout: 60000}).contains('Dashboard | EDX Automation Testing School');
```

Comment:
We want to keep timeouts to realistic lengths {like 3-5 seconds at most).  Could we lower these please?